mandatory/3-function-output.js

```diff
@@ -1,16 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
+
+  //returns a random number between 0 (included) and 1 (excluded) and multiple by 10:
```

Comment:
Do you know what kind of random number? Integer? Decimal? Negative or positive?

Reply:
It always returns a number lower than 1.

Reply:
Could it return -100? Or 0.56?